### ISSUE TITLE
"Stahovátko" uuid z klienta k následné práci v adminu

### ISSUE THREAD

**Issue by annie-cz** [ceskaexpedice/kramerius-web-client#639]
Bylo by super, aby vznikl kurátorský nástroj, který dovolí vyjet kurátorovi/adminovi libovolný seznam uuid (např. podle signatury) a zároveň aby bylo možné je nějak dál filtrovat, tj. např. když si vyjedu nějakou sadu dat v Krameriovi, pak bych překopírovala url řádek do nějakého nástroje a to by mi vyhodilo do tabulky seznam uuid, se kterým pak můžu pracovat (úplně nejlíp si to nějak modifikovat, jestli chci např. název, autora, rok vydání apod). Nebo aby to bylo schované jako další možnost pod "tužtičkou". Důležité je, aby to bylo ovladatelné z klienta, ne (jen) z admin rozhraní (byť se to týká admina a dat), protože ta vizuální kontrola nad sadou dat je pro kurátora důležitá. Člověk snadno u zadávání kritérií na slepo udělá nějakou chybu a takto by přesně viděl, co zadává jako seznam požadovaných uuid. A pak by se nebál je hromadně měnit.

Původně řešeno v rámci https://github.com/ceskaexpedice/kramerius-admin-client/issues/52.

**Comment by honza-rychtar** [comment]
Tohle je řešeno kombinací DEV módu, výběru počtu výsledků na stránku a stažení výsledků přes CSV.

**Comment by annie-cz** [comment]
Super, myslím, že můžeme zavřít. 🙂 Jen dotaz - je nějaký důvod, proč ten výběr počtu výsledků na stránku nemá i uživatel?

**Comment by honza-rychtar** [comment]
@annie-cz no já bych to vlastně nezavíral.
Tohle řešení je interní. Udělali jsem ho vlastně navíc a pro sebe, s tím že ho můžou používat i další, pokud chtějí.

> je nějaký důvod, proč ten výběr počtu výsledků na stránku nemá i uživatel?

Neručíme za to, že je to dokončené řešení vhodné do produkce pro uživatele. 
Pro naše účely je to ale zatím dostačující.

**Comment by zabak** [comment]
Souhlasím, udělejme pro to ale samostatné issue s definicí jak to  má vypadat a pak to tady znova zavřeme.

**Comment by annie-cz** [comment]
Ok, to dává smysl. Tak pokud se do budoucna bude přemýšlet nad tím, jak tohle udělat do budoucna lépe pro uživatele apod., tak to navrhuji rozdělit na dvě části:

- DevMode nacpat natvrdo do admin funkcí po přihlášení (stejně jako další kurátorské funkce, které tam už jsou a běžný čtenář je nevidí)
- Části DevMode, které mohou být vhodné už i pro uživatele (byť třeba až po přihlášení), přidat do klasického klienta - takže např. to stránkování

Co si o tom myslíte?

**Comment by zabak** [comment]
Ano. Pokud bys mohla založit dvě issue a vyjmenovat co za funkce spadá do kterého z nich, bylo by to ideální.

**Comment by annie-cz** [comment]
Done. Vznikly issues #769 a #770. Tady to tedy asi můžeme zavřít?
